### ISSUE TITLE
README for part #1 of the R1CS Tutorial is misguiding

### ISSUE THREAD

**Issue by fwmm** [arkworks-rs/r1cs-tutorial#11]
∂

## Summary of Bug

The README for the _merkle tree example_ in the R1CS tutorial incorrectly states that it will show how to invoke a SNARK when actually the example is only about generating the constraints and verifying satisfiability directly using the witness input itself. This is confusing for beginners trying to learn arkworks.

## Version

commit 5d3a9022fb6deade245505748fd661278e9c0ff9

## Steps to Reproduce

Compare the README to the example code.

**Comment by neocarmack** [comment]
yes. The example is simply filling the merkle path to verify, it is not a construction of r1cs. The knowledge is exposed, not a zero knowledge proof.
